code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -131,11 +131,13 @@ var/list/wizard_snakes
 		wizard_snakes = list()
 	wizard_snakes.Add(src)
 
-/mob/living/simple_animal/cat/snek/wizard/Die()
+/mob/living/simple_animal/cat/snek/wizard/death_untransmogrify()
 	if(!transmogrify())
 		visible_message("<span class='notice'>\The [src] vanishes!</span>")
 		qdel(src)
```

Comment:
There should probably be a `return FALSE` right here under the `qdel()` to signify that the wizard snake was not untransmogrified.